src/ts/ssh-tcp/services/portForwardingService.ts

```diff
@@ -882,6 +889,7 @@ export class PortForwardingService extends SshService {
 		port: number,
 		cancellation?: CancellationToken,
 	): Promise<SshChannel> {
+		this.raiseReportProgress(Progress.OpeningChannelPortForwarding);
```

Comment:
I'd suggest OpeningChannelForForwardedPort or OpeningPortForwardingChannel. And similar for Opened below.